submitter/sample_data.py

```diff
@@ -0,0 +1,117 @@
+import json
+
+import boto3
+
+
+def sample_data_loader(id, source, target, col_hdl, meta_loc, filename, fileloc):
+    sqs = boto3.resource("sqs")
+    queue = sqs.get_queue_by_name(QueueName="queue1-stage")
+    body = {
+        "SubmissionSystem": target,
+        "CollectionHandle": col_hdl,
+        "MetadataLocation": meta_loc,
+        "Files": [
+            {"BitstreamName": filename, "FileLocation": fileloc},
+        ],
+    }
+    # Send message to SQS queue
+    queue.send_message(
+        MessageAttributes={
+            "PackageID": {"DataType": "String", "StringValue": id},
+            "PackageSource": {"DataType": "String", "StringValue": source},
+        },
+        MessageBody=(json.dumps(body)),
+    )
+
+
+def sample_data():
```

Comment:
This has 10 messages, so we know it works with our max number of messages. Have we tried it with 20? Can we?

Reply:
the max value is really a rough estimate in SQS... it essentially returns between 0 and max messages. With long polling it _usually_ returns max messages but not always. There is an edge case in which it will return 0 and then we'll stop processing thinking we finished but we haven't. We'll have to look into how to handle this if we notice it in production... so far long polling has prevented me from seeing this edge case but I'm sure it'll creep up when we least expect it.